evolver/calibration/actions.py

```diff
@@ -0,0 +1,88 @@
+from typing import Dict, Any
+from copy import deepcopy
+
+
+class DisplayInstructionAction:
```

Comment:
should we put an action interface to formalize things, like:


```py
class CalibrationAction(ABC):
   @abstracmethod
   def execute(): ...
```

Reply:
ah nice idea!